### ISSUE TITLE
Suggestion: add descriptive tooltip to "1's 2's 3's" in XSheet toolbar

### ISSUE THREAD

**Issue by Cyberman-tM** [opentoonz/opentoonz#3098]
![grafik](https://user-images.githubusercontent.com/4354369/74107173-c5c8a180-4b6d-11ea-8d15-4dc544caf6e0.png)
As you can see the tooltip for the button "1's" merely displays the same as the button. Leaving newbs like me to wonder what exactly this does.

### System Information
1.4
Windows

**Comment by gab3d** [comment]
you're right, the current one provides no additional help.

**Comment by DarrenTAnims** [comment]
This is fairly easy to change. But what would you want the text to be? I've got some thoughts, but it would be easy to change it and the change to not be very clear or too wordy when trying to make it clear. My initial thoughts are:

1's: Changes the repetition of the selected drawings so that every unique drawing is shown for 1 frame.
2's: Changes the repetition of the selected drawings so that every unique drawing is shown for 2 frames.
3's: Changes the repetition of the selected drawings so that every unique drawing is shown for 3 frames.
4's: Changes the repetition of the selected drawings so that every unique drawing is shown for 4 frames.

Do you have a better description?

**Comment by artisteacher** [comment]
Maybe something shorter:
Show selected drawings for 1 frame each
Show selected drawings for 2 frames each...

**Comment by manongjohn** [comment]
How about something like: Reframe selection on X's

**Comment by gab3d** [comment]
ermmm... I would try to go with a shorter description, for an in depth one users should better resort to the manual. 🤔 

1's: Expose drawings in selected cells on ones.
2's: Expose drawings in selected cells on twos.
3's: Expose drawings in selected cells on threes.
4's: Expose drawings in selected cells on fourths.

**Comment by Cyberman-tM** [comment]
Maybe i'm wrong - but isn't this called "refactoring" elsewhere? There's an option in the context menu for it I think.
If so - "Refactor using 1",  etc...
Doesn't fully EXPLAIN it, but it gives you more to search for than "button that says 1's".

**Comment by gab3d** [comment]
On a quick search, I don't see it called "refactoring" in any of the popular software I know of (Toon Boom, CSP, Adobe Animate, After Effects) 🤔

**Comment by Cyberman-tM** [comment]
Huh. Weird, did I dream that? I'm sure I read it somewhere :-/
Goes to show - think twice before you write write. Or something like that...

**Comment by gab3d** [comment]
...easy, no prob 😸

**Comment by manongjohn** [comment]
These options normally are found under main menu `Cells` -> `Reframe` - > `1's` , etc hence my suggestion being `Reframe selection on X's`

**Comment by gab3d** [comment]
yes, I know. it's ok...
I said "expose" instead of "reframe" mainly as a way of explaining it a little more, other than just repeating the term found in the menus. 🤷‍♂️

(also, to me at least, reframe is a little ambiguous term, as it could also be referring to a change in the camera framing, for example, or a change in keyframing... well, you get the idea).

**Comment by Lovewyrm** [comment]
I would say that ambiguity in this case should be cleared by reading into 'lingo of the trade' of animation.

Lightbox, and especially onion skin have to be understood as part of the trade, unless you want a tooltip that explains them in a lengthy manner.

Reanimate on 1's 2's' 3's.
I'm animating on ones.
I'm animating on twos.

Is what I would call it, and even that is weak.
Reframing and exposing for are weak too.
Everything is weak because this is something an animator know without a tooltip.

**Comment by Cyberman-tM** [comment]
Anything that gives something to look for should be sufficient IMO. The current tooltip isn't.

**Comment by Lovewyrm** [comment]
Well, if it's really supposed to be for newbies, then go all out and make it wordy.

"Click this to space the selected cells to one drawing every frame, without holds inbetween"
"Click this to space the selected cells to one drawing every two frames, with holds between them"
"Click this to space the selected cells to one drawing every three frames, with holds between them"

**Comment by RodneyBaker** [comment]
I agree that most users shoudn't need the tooltip.
As we are here to resolve that lacking detail however...

Most tooltips are from one to three words in length.
A four word tooltip works also but is generally considered long/lengthy.

The two words that fit closest are 'reframe' and 'expose'.
Reframe is the action the user is accomplishing when they use the process. (i.e. reframe from 1s to 2s, etc)
Expose is the proper term for what is occurring. (i.e. we are exposing images for 1, 2, 3 frames, etc)
While 'animating on 2s, etc. is a proper term it doesn't fit as well in this context because this is exposures on a exposure sheet and not 'animating on an exposure sheet'.  If the icons tooltips were seen in the viewer 'Animate on 2's', etc. would work.

As far as I'm concerned all three icon's tooltips could say, 'Reframe Exposures'**
Notice the two keywords 'frame' and 'expose' in that.
A lengthy toodtip customized for each then would be 'reframe exposures (1's)' although the appended number isn't necessary because the user can see that number in the icon.

UI discussions are always fun.  :)

**Note that these terms do require the inquisitive user/aspiring animator to know (or learn) what reframing and exposing is, first by exposure to the terms (hence the tooltip) and then increasingly by experience of what clicking on the icon actually does.

**Comment by Lovewyrm** [comment]
Yeah, UI stuff is fun/tricky when it's cases like these. =P

Maybe "Read the manual" would be the most honest¹, if the tooltip doesn't explain what it does in a way that stands on its own.
I mean, I guess it's fine on something benign like this, but on a principle, and all that?

If it's not obvious then it requires experimentation, and that's not as much fun as knowing what's about to happen.

¹I mean, that might come across as rude or 'cop out' but, again, if the tooltip doesn't convey what it does without having to experient/read the manual anyway...well...then what's the point?

**Comment by beeheemooth** [comment]
For "Pioneer plaques"

|1|2|3|4|..............
|1|.|2|.|3|.|4|........
|1|..|2|..|3|..|4|.....
|1|...|2|...|3|...|4|..
|1|....|2|....|3|....|4|

**Comment by Cyberman-tM** [comment]
> Maybe "Read the manual" would be the most honest¹, if the tooltip doesn't explain what it does in a way that stands on its own.
What do you look for when all you have to look for is "1's" ?

**Comment by Lovewyrm** [comment]
The manual. The manual is there even before I run the program.

**Comment by Cyberman-tM** [comment]
forgot to add: OK, forget it.

**Comment by gab3d** [comment]
May I lastly add that the increased use of the words "exposure", "expose", etc. gives new users more meaning as yo why they are really using an "eXposure Sheet", in first place. 🤔

**Comment by manongjohn** [comment]
Not sure why you closed it. I think it would be good to be fixed for clarity sake. Granted, there are probably a hundred more places where tool tips should be fixed/clarified.

As far as wording goes, I would keep to a short/simple description just as a reminder of what it is for. It's a quick "tip", not a book.  Try to use industry standard terminology when possible.

I find extremely wordy tool tips to be obnoxious, especially if I am hovering over something and accidentally trigger a tool tip which then throws a huge blob of text at me.

If a user requires more information they should go to the manual, which is easier to get to now that there is a link to it in the Help menu.  If they aren't sure about terminology, hopefully they know to search on-line.

**Comment by gab3d** [comment]
@manongjohn 
I agree, short is good.

**Comment by DarrenTAnims** [comment]
Agreed here too. "Reframe on 1's" would be perfectly fine in my mind. This doesn't need closing though. It's a quick fix and would help new users.
